src/express.js

```diff
@@ -0,0 +1,19 @@
+const getPublicKey = require('./public-key');
+const validateWebhook = require('./validate-webhook');
+
+function eventbusMiddleware(req, res, next) {
+    const token = req.get('x-eventbus-authentication');
+
+    if (!token)
+        return next(new Error('No valid EventBus token found'));
+
+    if (!req.body)
+        return next(new Error('Unable to find body, use body-parser'));
+
+    getPublicKey()
+        .then(publicKey => validateWebhook(publicKey, token, req.body))
+        .then(() => next())
```

Comment:
This should probably be `.then(() => next, (err) => next(err))` otherwise an error in `next` might cause `next` to be called twice from the middleware.